ibtk/include/ibtk/KrylovLinearSolverPoissonSolverInterface.h

```diff
@@ -68,6 +68,12 @@ class KrylovLinearSolverPoissonSolverInterface : public PoissonSolver
      */
     void setPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& poisson_spec) override;
 
+    /*!
+     * \brief Set the IBTK::ProblemSpecification object used to specify
+     * the coefficients for the scalar-valued or vector-valued Laplace operator.
+     */
+    void setProblemSpecification(const IBTK::ProblemSpecification* problem_spec) override;
```

Comment:
Here I think you can see the problem with having both `setPoissonSpecifications` and `setProblemSpecification` --- I think we could have one or the other but should not use both in the same interface.

Reply:
@boyceg: Based on these three PRs that require setting `ProblemSpecification` which derived class should I delegate `setProblemSpecification()` to?

or are you recommending removing `setPoissonSpecifications` entirely? For the latter would you want to re-design the interfaces properly? I can follow your lead and rebase PR1 (and others) on it.